geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/GeolocationPlugin.java

```diff
@@ -0,0 +1,201 @@
+package com.capacitorjs.plugins.geolocation;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+import com.getcapacitor.annotation.CapacitorPlugin;
+import com.getcapacitor.annotation.Permission;
+import java.util.HashMap;
+import java.util.Map;
+
+@CapacitorPlugin(
+    name = "Geolocation",
+    permissions = {
+        @Permission(permission = Manifest.permission.ACCESS_COARSE_LOCATION, alias = "location"),
+        @Permission(permission = Manifest.permission.ACCESS_FINE_LOCATION, alias = "location")
+    },
+    permissionRequestCode = GeolocationPlugin.GEOLOCATION_REQUEST_PERMISSIONS
+)
+public class GeolocationPlugin extends Plugin {
+
+    public static final int GEOLOCATION_REQUEST_PERMISSIONS = 7179;
```

Comment:
Was `7179` copied from somewhere? I think we should try to keep the official request codes in the 90xx range.

Reply:
No I changed it from the original code so it didn't conflict with the codes in the core. I can change this to be back in the 90xx range if that's what we want